datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/exception/DataHubGraphQLErrorCode.java

```diff
@@ -4,6 +4,7 @@ public enum DataHubGraphQLErrorCode {
   BAD_REQUEST(400),
   UNAUTHORIZED(403),
   NOT_FOUND(404),
+  UNAUTHORIZED_TAG_ERROR(405),
```

Comment:
HTTP 405 is well defined in OSS. Let's not override the standard HTTP codes

Reply:
Reverted back the error code to 403 with different error message